Source/Projections/Store/NoTypeAssociatedWithProjection.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.SDK.Events;
+
+namespace Dolittle.SDK.Projections.Store
+{
+    /// <summary>
+    /// Exception that gets thrown when there is no <see cref="Type" /> associated with a projection.
+    /// </summary>
+    public class NoTypeAssociatedWithProjection : Exception
```

Comment:
Why would this happen? I don't see where we would resolve the type from an projection ID right now.

Reply:
I guess that this was put in for consistency. We could remove the possibility of resolving type from id